src/Microsoft.AspNet.Hosting/HostingServices.cs

```diff
@@ -17,16 +17,19 @@ public static class HostingServices
         private static IServiceCollection Import(IServiceProvider fallbackProvider, Action<IServiceCollection> configureHostServices)
         {
             var services = new ServiceCollection();
-            if (configureHostServices != null)
-            {
-                configureHostServices(services);
-            }
             var manifest = fallbackProvider.GetRequiredService<IServiceManifest>();
             foreach (var service in manifest.Services)
             {
                 services.AddTransient(service, sp => fallbackProvider.GetService(service));
             }
+
```

Comment:
odd, it's working for the MVC scenarios and that involves not only replacing but also adding services.  and `AddHosting` doesn't replace the manifest either.  is the `IServiceManifest` needed?